src/openfermion/ops/__init__.py

```diff
@@ -21,3 +21,4 @@
 from ._interaction_operator import InteractionOperator
 from ._interaction_rdm import InteractionRDM
 from ._quadratic_hamiltonian import QuadraticHamiltonian
+from ._binary_operator import SymbolicBinary
```

Comment:
we prefer to import things in alphabetical order, meaning this should be the first import. However, sometimes there is a problem with circular imports. If that is the case here, then it is fine.